editor/components/inserter/test/menu.js

```diff
@@ -107,12 +139,21 @@ describe( 'InserterMenu', () => {
 			/>
 		);
 
-		const activeCategory = wrapper.find( '.components-panel__body.is-opened > .components-panel__body-title' );
-		expect( activeCategory.text() ).toBe( 'Most Used' );
+		const activeCategory = wrapper.root.find( ( node ) => {
+			return node.props.className &&
+				node.props.className.includes( 'components-panel__body is-opened' );
+		} ).find( ( node ) => {
+			return node.props.className &&
+				node.props.className.includes( 'components-panel__body-title' );
+		} ).find( ( node ) => {
+			return node.type === 'button';
```

Comment:
Oof, this is a fair bit of effort to find a descendent node. I wonder if we could do for some of our small abstractions for common use cases.

Reply:
Another option here would be to convert the component to a DOM node and then use `querySelector` on it.  That will at least reduce this traversal to a single selector string similar to what was originally used.